floodviz/static/js/hydrograph.js

```diff
@@ -143,39 +141,43 @@
 				update(data);
 			});
 		};
-    
+
 		/**
-		 * Handle all mouse over movements for calling element. 
+		 * Displays tooltip for hydrograph at a datapoint in addition to
+		 * corresponding map site tooltip
 		 */
-		self.mouseover = function(d) {
+		self.series_tooltip_show = function(d) {
 			d3.select(d.data.name).classed("gage--hover", true);
 			focus.attr("transform", "translate(" + x(d.data.time_mili) + "," + y(d.data.value) + ")");
 			focus.select("text").html(d.data.key + ": " + d.data.value + " cfs " + " " + d.data.time + " " + d.data.timezone);
-			// Interative linking with map
-			FV.map_figure.mousemove(d.data.name, d.data.key);
+			// Interactive linking with map
+			FV.map_figure.site_tooltip_show(d.data.name, d.data.key);
```

Comment:
So ask yourself does FV.map_figure.site_tooltip_show call FV.hydrograph_figure.series_tooltip_show? If it does we have an infinite loop. And secondly, you want to keep this function  simple ... it should only update the hydrograph. You should not be using FV.map_figure directly in this function. Instead pass in a function to do any additional processing that needs to happen outside of the hydrograph.